server/controllers/routes/teacherProfile.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+const { getTeacherProfile } = require('../../database/quieres/teacherProfile');
+
+require('env2')('./config.env');
+
+const secret = process.env.SECRET_KEY;
+
+const teacherProfile = (req, res, next) => {
+  getTeacherProfile()
+    .then(({ rows }) => {
+      if (req.cookies.access) {
+        jwt.verify(req.cookies.access, secret, (error, value) => {
+          if (value) {
+            res.json(rows);
+          } else {
+            res.redirect('/');
```

Comment:
why redirect? This is API call, not the browser..

Reply:
thanks, I got you and I edited it